lib/drivers/tandemDriver.js

```diff
@@ -1801,14 +1802,31 @@ module.exports = function (config) {
       if(alarmEntry.alarm_id != null) {
         var alarmValue = alarmEntry.alarm_id;
         var alarmText = getAlarmName(alarmValue,ALARM_TYPES);
+        var postbasal = null;
 
         switch (alarmValue) {
           case ALARM_TYPES.ALARM_OCCLUSION.value:
             alarmRecord = alarmRecord.with_alarmType('occlusion');
```

Comment:
As per our discussion: Since they should be infrequent, I'm going to annotate these suspended basals as `tandem/basal/fabricated-from-occlusion-alarm`.